### PR TITLE
chore(CHANGELOG): add changelog for v0.4.9

### PR DESCRIPTION
**Pull request by akhilerm**
Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Comment by akhilerm** [comment]
@shovanmaity  can you review the changes in CHANGELOG corresponding to the fixes you have done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+  * enable persisting of labels and annotations on BlockDevice resource ([#394](https://github.com/openebs/node-disk-manager/pull/394), 
+  [shovanmaity](https://github.com/shovanmaity))
+  * add label selector to BlockDeviceClaim resource ([#397](https://github.com/openebs/node-disk-manager/pull/397), 
+  [@akhilerm](https://github.com/akhilerm))
+  * add support for `openebs.io/block-device-tag` label on BlockDevice ([#400](https://github.com/openebs/node-disk-manager/pull/400), 
+  [@akhilerm](https://github.com/akhilerm))
+
+v0.4.9-RC2 / 2020-04-11
+========================
+  * No changes since RC1
```
Comment (akhilerm) [inline CHANGELOG.md:17]: @kmova Is this actually required?. IMO, we should not keep a section just for the title.

Comment (kmova) [reply CHANGELOG.md:17]: Dont' have a strong opinion on this one. Remove the pr/hold-merge, after you have resolved this comment with either removing it or keep it. Either way is fine for me.

Comment (akhilerm) [reply CHANGELOG.md]: Removed the RC2 title.

**Pull request merged**